src/main/java/gnu/io/RXTXPort.java

```diff
@@ -997,7 +957,8 @@ public void notifyOnBreakInterrupt( boolean enable )
 
     public void close()
 	{
-                log.trace( "RXTXPort:close( " + this.name + " )"); 
+				log.trace( "RXTXPort:close( " + this.name + " )");
```

Comment:
Think you probably meant to remove this trace, not reindent it.

Reply:
Yes, it's a tabs vs spaces issue.